src/yaml.rs

```diff
@@ -310,8 +310,9 @@ impl Yaml {
         }
         match v {
             "~" | "null" => Yaml::Null,
-            "true" => Yaml::Boolean(true),
-            "false" => Yaml::Boolean(false),
+            // http://yaml.org/type/bool.html
+            "true" | "True" | "TRUE" | "yes" | "Yes" | "YES" | "on" | "On" | "ON" => Yaml::Boolean(true),
```

Comment:
this would introduce a regression, your [link](http://yaml.org/type/bool.html) points to the 1.1 spec, but this is supposed to be a yaml1.2 library. In 1.2 all of these keywords nolonger become bools but strings.
http://www.yaml.org/spec/1.2/spec.html#id2803629

Reply:
unless you want to add an explicit `1.1` mode perhapse, yaml has the attribute `%Yaml 1.1` in line 1 before `---`